fe/fe-core/src/main/java/com/starrocks/qe/Coordinator.java

```diff
@@ -693,19 +741,19 @@ private void deliverExecFragmentRequests(boolean enablePipelineEngine) throws Ex
      * - Each group should be delivered sequentially, and fragments in a group can be delivered concurrently.
      * <p>
      * For example, the following tree will produce four groups: [[1], [2, 3, 4], [5, 6], [7]]
-     *         1
-     *         │
-     *    ┌────┼────┐
-     *    │    │    │
-     *    2    3    4
-     *    │    │    │
+     * 1
+     * │
+     * ┌────┼────┐
+     * │    │    │
+     * 2    3    4
+     * │    │    │
      * ┌──┴─┐  │    │
      * │    │  │    │
      * 5    6  │    │
-     *      │  │    │
-     *      └──┼────┘
-     *         │
-     *         7
+     * │  │    │
+     * └──┼────┘
+     * │
+     * 7
```

Comment:
Formatting this code destroys the ASCII graph.

Reply:
done